spec/bot.intergration.ts

```diff
@@ -64,18 +65,19 @@ const mockAnalyzerClient: AnalyzerClient = {
   }
 };
 
+const repliers = [
+  docsClient,
+  eventsClient,
+]
 const analyzers = [mockAnalyzerClient];
 
-commandTree.registerSubCommand(docsClient.commandSubtree);
-commandTree.registerSubCommand(eventsClient.commandSubtree);
-
 const dummyMessage: MessageModel = {
   text: 'some text here'
 };
 
-describe(`Angie`, () => {
+describe(`Bot`, () => {
 
-  const angie = new Angie(null, null, true, commandTree, analyzers, 0, null, null);
+  const angie = new Bot(null, null, false, repliers, analyzers);
```

Comment:
It annoys me aswell but havn't decided what to do about it jsut yet on how to fix it up, preferably we should not havn't to pass any config like that since it's just silly